lib/listening.js

```diff
@@ -1,17 +1,21 @@
 // Native
+const { existsSync, statSync } = require('fs')
 const path = require('path')
 
 // Packages
 const ignoredDirectories = require('ignore-by-default').directories()
```

Comment:
Still need to copy the array returned from `directories`

Reply:
I don't understand, why?

Reply:
Cause it's a list of 4 items, so it's not really needed as a dependency, we can just keep track of them in micro-dev 👍

Reply:
@timneutkens Is this PR ready to merge?